### DISCUSSION TITLE
Implement map helper function

### DISCUSSION THREAD

**Discussion by believelody** [handlebars-lang/handlebars.js#1937] [Q&A]
Hi guys, I'm trying to implement a kind of `map` helper similar to javascript map function. So my question is how can i pass a function to an helper ?
Ideally to have something like: `(map array double)` where `double` is the function which will take each array's element.

**Comment by jaylinski** [comment]
You can take inspiration from `map`-helper in the handlebars-helpers package:

https://github.com/helpers/handlebars-helpers/blob/5d3405f178a9ec4c90df3c7dc07b4faf9a09dfd7/lib/array.js#L397-L428
